Add flag for latest version (#316)
NethermindEth/sedge: docs/docs/quickstart/complete-guide.mdx

:::

> :::info
> 
> If you want to use the latest version of the clients, and don't use the fixed image tested by us, you can generate the
> setup using latest version of them using the following command: `sedge generate full-node --latest`. If you want to learn
> more, you can check [here in our documentation](/docs/commands/generate).
> 
> You can use `--latest` flag with `full-node`, `execution`, `consensus` and `validator` commands.
> 
> If the clients release any version, you can just:
> 1. Run `docker compose -f /path/to/docker-compose.yml pull` to update the images.
> 2. Run `docker compose -f /path/to/docker-compose.yml up -d` to restart the containers.
> 
> :::
> 

Reviewer: ```suggestion
:::info

If you want to use the client's latest version and avoid using the fixed version tested by us, you can generate the
setup using the following command: `sedge generate full-node --latest`. If you want to learn
more, you can check [here in our documentation](/docs/commands/generate).

The `--latest` flag can be used for the `full-node`, `execution`, `consensus` and `validator` commands.

You can update your client's versions using the following commands if the clients release any version:
1. Run `docker compose -f /path/to/docker-compose.yml pull` to update the images.
2. Run `docker compose -f /path/to/docker-compose.yml up -d` to restart the containers.

:::

```
